.github/workflows/ios_e2e.yml

```diff
@@ -37,3 +37,4 @@ jobs:
             ## Flashlight iOS Report 🔦
 
             - [🚀 Report](${{ env.NETLIFY_PREVIEW_URL }})
+          append: true
```

Comment:
why the append? I think by default it edits previous message

Reply:
I thought it would be better because now we compare with the previous run. I can remove it

Reply:
Removed